CHANGELOG.md

```diff
@@ -6,25 +6,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.0] - 2019-04-05
+
 ### Added
-* `demos/usage-dag-edges.py`: Show different types of edges in a DAG
-* `demos/usage-elements-extra.py`: Shows how to load external layouts, otherwise same app as `usage-elements.py`.
-* `demos/usage-preset-animation.py`: Example of animating nodes using the preset layout.
-* `demos/usage-reset-button.py`: Example of resetting the graph position using a button.
-* `dash_cytoscape.load_extra_layouts()`: A new function that can be called before initializing the Dash app (`app = dash.Dash(__name__)`) to load the JS bundle containing the external layouts. 
-* `webpack.[dev|prod].extra.config.js`: Two new webpack configs for external layouts.
-* `src/lib/extra_index.js`: Loads external layouts before exporting the `Cytoscape` class. Needed to generate the new bundles.
-* Images of new external layouts.
+* Four new demos:
+    * `demos/usage-dag-edges.py`: Show different types of edges in a DAG. It uses the new `dash_cytoscape.utils.Tree` class.
+    * `demos/usage-elements-extra.py`: Shows how to load external layouts, otherwise same app as `usage-elements.py`.
```

Comment:
```suggestion
    * `demos/usage-elements-extra.py`: Shows how to load external layouts.
```

I don't think you have to specify the similarity to `usage-elements`, but feel free to leave it in if you want!